proposals/collection-literals.md

```diff
@@ -243,6 +235,10 @@ var e2 = [..e1];           // List<object> or error?
 List<string> e3 = [..e1];  // error?
 ```
 
+```c#
+var c = [..[1, 2, 3]]; // List<int>
+```
```

Comment:
At this position in the file, this looks like it falls under the text "Should `IEnumerable` contribute an *iteration type* of `object` or no contribution?" but I didn't see the connection.

Reply:
Added comment.